recipes/pyusb/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pyusb" %}
+{% set version = "1.0.2" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "4e9b72cc4a4205ca64fbf1f3fff39a335512166c151ad103e55c8223ac147362" %}
+{% set build_number = 0 %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: {{ build_number }}
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - usb
+    - usb.backend
+
+about:
+  home: http://walac.github.io/pyusb
```

Comment:
I think the following values should be changed:
home: https://github.com/pyusb/pyusb
license: Apache-2.0
license_family: Apache
doc_url: https://github.com/pyusb/pyusb/blob/master/docs/tutorial.rst